client/src/config/wagtailConfig.js

```diff
@@ -31,3 +15,18 @@ function getWagtailConfig() {
 }
 
 export const WAGTAIL_CONFIG = getWagtailConfig();
```

Comment:
We can also change this file to use TypeScript and add the interface definition to this `WAGTAIL_CONFIG` object, so other modules that import this object will be able to get the typings.